qabel_provider/models.py

```diff
@@ -25,6 +25,10 @@ class Profile(models.Model, ExportModelOperationsMixin('profile')):
     pro_notification_mail = models.BooleanField(default=False)
 
     @property
+    def is_disabled(self):
+        return not self.is_allowed() and self.user.is_active
```

Comment:
is_allowed already checks for user.is_active
and didn't you mean `not (self.is_allowed() and self.user.is_active)`?